### PR TITLE
python3Packages.python-gvm: init at 21.6.0, python3Packages.gvm-tools: init at 21.6.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Collection of APIs that help with remote controlling a Greenbone Security Manager

https://github.com/greenbone/python-gvm

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 137031` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.python-gvm</li>
    <li>python39Packages.python-gvm</li>
  </ul>
</details>

**Review by samuela — APPROVED** [review]

File: pkgs/development/python-modules/python-gvm/default.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "070qpj2y7834i50lhkkbv93s77j91js06zs1bpbmplppiraxqmyz";
+  };
+
+  nativeBuildInputs = [
+    poetry
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-gvm/default.nix:27]: not poetry-core?

**Comment by risicle** [comment]
On macos 10.15:

```
__________________ GvmConnectionTestCase.test_feed_xml_error ___________________

self = <tests.connections.test_gvm_connection.GvmConnectionTestCase testMethod=test_feed_xml_error>

    def test_feed_xml_error(self):
        connection = GvmConnection()
        connection._start_xml()
        with self.assertRaises(
            GvmError, msg='Cannot parse XML response. Response data read bla'
        ):
>           connection._feed_xml("bla")

tests/connections/test_gvm_connection.py:58: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gvm/connections.py:83: in _feed_xml
    self._parser.feed(data)
src/lxml/parser.pxi:1256: in lxml.etree._FeedParser.feed
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   lxml.etree.ParserError: Unicode parsing is not supported on this platform

src/lxml/parser.pxi:1291: ParserError

```

**Comment by risicle** [comment]
Opened https://github.com/greenbone/python-gvm/issues/550

This one looks like it's just an oversight in the test itself, so it should be fine to just skip this test on darwin.

**Comment by fabaff** [comment]
> This one looks like it's just an oversight in the test itself, so it should be fine to just skip this test on darwin.

Excluded on darwin.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 137031` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>gvm-tools (python39Packages.gvm-tools)</li>
    <li>python38Packages.gvm-tools</li>
    <li>python38Packages.python-gvm</li>
    <li>python39Packages.python-gvm</li>
  </ul>
</details>

**Pull request merged**